src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -72,6 +91,33 @@ class PalantirDockerPlugin implements Plugin<Project> {
                 workingDir dockerDir
                 commandLine 'docker', 'push', ext.name
             }
+
+            dockerfileZip.with {
+                from(ext.resolvedDockerfile)
+            }
+
+            // Configure docker-compose templating
+            if (ext.resolvedDockerComposeTemplate.exists()) {
```

Comment:
I think there likely needs to be two states here -- if you configure the template file but it doesn't exist, we likely need to error so that users get reasonable feedback